src/main/java/com/hcl/appscan/jenkins/plugin/builders/AppScanEnterpriseBuildStep.java

```diff
@@ -788,6 +974,50 @@ public FormValidation doCheckJobName(@QueryParameter String jobName) {
 			return FormValidation.validateRequired(jobName);
 		}
 
+		public FormValidation doCheckExploreDataTestScan(@QueryParameter String exploreDataTestScan) {
+			return FormValidation.validateRequired(exploreDataTestScan);
+		}
+
+		public FormValidation doCheckPostmanCollectionFile(@QueryParameter String postmanCollectionFile) {
+			if(postmanCollectionFile != null && !postmanCollectionFile.isEmpty()) {
+				if (!postmanCollectionFile.endsWith(".json")) {
+					return FormValidation.error(Messages.error_file_type_invalid_json());
+				}
+			}
+			return FormValidation.validateRequired(postmanCollectionFile);
+		}
+
+		public FormValidation doCheckAdditionalDomains(@QueryParameter String additionalDomains) {
+			return FormValidation.validateRequired(additionalDomains);
+		}
+
+		public FormValidation doCheckEnvironmentalVariablesFile(@QueryParameter String environmentalVariablesFile) {
+			if(environmentalVariablesFile != null && !environmentalVariablesFile.isEmpty()) {
+				if (!environmentalVariablesFile.endsWith(".json")) {
+					return FormValidation.error(Messages.error_file_type_invalid_json());
+				}
+			}
+			return FormValidation.ok();
+		}
+
+		public FormValidation doCheckGlobalVariablesFile(@QueryParameter String globalVariablesFile) {
+			if(globalVariablesFile != null && !globalVariablesFile.isEmpty()) {
+				if (!globalVariablesFile.endsWith(".json")) {
+					return FormValidation.error(Messages.error_file_type_invalid_json());
+				}
+			}
+			return FormValidation.ok();
+		}
+
+		public FormValidation doCheckAdditionalFiles(@QueryParameter String additionalFiles) {
+			if(additionalFiles != null && !additionalFiles.isEmpty()) {
```

Comment:
Rather than a nested "if" block, this should just be logically "anded" with the other conditions.